controlplane/api/connection/connection.proto

```diff
@@ -34,13 +35,24 @@ enum State {
   DOWN = 1;
 }
 
+message PathSegment {
+  string name      = 1;
+  string id        = 2;
+  string token = 3;
```

Comment:
@edwarnicke How is generating this token, is ever NSE could add some data encoded with it? For example if NSE is passing to another network service, and it would be good to have endpoint-id to be encoded here to be able to restore full sequence on same NSEs instead of new ones.